components/table/stories/table.stories.js

```diff
@@ -125,25 +138,156 @@ export default {
+	},
+	{
+		cellContent: "Table Row Delta",
+		showCheckbox: true,
+	},
+	{
+		cellContent: "Echo",
+		showCheckbox: true,
+	},
+	{
+		cellContent: "Foxtrot",
+		showCheckbox: true,
+	},
+];
+
+/**
+ * The medium size is the default and recommended option.
+ */
 export const Default = TableGroup.bind({});
 Default.args = {};
 
-// ********* VRT ONLY ********* //
-export const WithForcedColors = TableGroup.bind({});
-WithForcedColors.tags = ["!autodocs", "!dev"];
-WithForcedColors.parameters = {
-	chromatic: {
-		forcedColors: "active",
-		modes: disableDefaultModes,
-	},
+// ********* DOCS ONLY ********* //
+/**
+ * Tables come in four different sizes: small, medium, large, and extra-large. The medium size is the default and recommended option.
+ */
+export const Sizing = (args, context) => Sizes({
+	Template,
+	withHeading: false,
+	withBorder: false,
+	...args,
+}, context);
+Sizing.args = {};
+Sizing.tags = ["!dev"];
+Sizing.parameters = {
+	chromatic: { disableSnapshot: true },
 };
 
-// ********* DOCS ONLY ********* //
 /**
- * An example showing both the optional summary row, and a row marked as selected.
+ * The compact variant decreases the spacing used within the table.
  */
-export const SummaryAndSelected = Template.bind({});
+export const Compact = Template.bind({});
+Compact.args = {
+	...Default.args,
+	density: "compact",
+};
+Compact.tags = ["!dev"];
+Compact.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The spacious variant increases the spacing used within the table.
+ */
+export const Spacious = Template.bind({});
+Spacious.args = {
+	...Default.args,
+	density: "spacious",
+};
+Spacious.tags = ["!dev"];
+Spacious.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The standard multi-select table includes a column of checkboxes used for selecting rows.
+ */
+export const MultiSelect = Template.bind({});
+MultiSelect.storyName = "Multi-select";
+MultiSelect.args = {
+	rowItems: ExampleRowItems,
+};
+MultiSelect.tags = ["!dev"];
+MultiSelect.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * Excluding the `.spectrum-Table--emphasized` class will change the style of selected rows.
+ */
+export const NonEmphasizedMultiSelect = Template.bind({});
+NonEmphasizedMultiSelect.storyName = "Non-emphasized multi-select";
+NonEmphasizedMultiSelect.args = {
+	...MultiSelect.args,
+	isEmphasized: false,
+};
+NonEmphasizedMultiSelect.tags = ["!dev"];
+NonEmphasizedMultiSelect.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * The quiet table has a transparent background and no borders on the left and right.
+ */
+export const Quiet = Template.bind({});
+Quiet.args = {
+	...Default.args,
+	isQuiet: true,
+};
+Quiet.tags = ["!dev"];
+Quiet.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const QuietMultiSelect = Template.bind({});
+QuietMultiSelect.args = {
+	...MultiSelect.args,
+	isQuiet: true,
+};
+QuietMultiSelect.tags = ["!dev"];
```

Comment:
It might be that it's almost the end of the day and my brain is melting from looking at all these table variants, but I definitely looked at the docs site and was confused as to why a variant with "Quiet" in it had emphasized coloring. I think it's a little clearer with the ordering of the stories here and I get it now, but it wouldn't be a terrible idea to leave a comment here about how this is both quiet and emphasized, or it's quiet but you can also leave off the emphasized modifier class here if you want. Or you can leave it as is if you hate all those ideas 🙂

Reply:
How's this? https://github.com/adobe/spectrum-css/pull/2925/commits/1e5cf20f050ea6d33feb1afa6c8f231faccbfe33

It should mimic the non-emphasized story description now.